sklearn/tests/test_calibration.py

```diff
@@ -275,3 +277,29 @@ def test_calibration_nan_imputer():
     clf_c = CalibratedClassifierCV(clf, cv=2, method='isotonic')
     clf_c.fit(X, y)
     clf_c.predict(X)
+
+
+def test_calibration_prob_sum():
+    # Test that sum of probabilities is 1. A non-regression test for
+    # issue #7796
+    num_classes = 2
+    X, y = make_classification(n_samples=10, n_features=5,
+                               n_classes=num_classes)
+    clf = LinearSVC(C=1.0)
+    clf_prob = CalibratedClassifierCV(clf, method="sigmoid", cv=LeaveOneOut())
+    clf_prob.fit(X, y)
+
+    probs = clf_prob.predict_proba(X)
+    assert_array_almost_equal(probs.sum(axis=1), np.ones(probs.shape[0]))
+
+    # Test to check calibration works fine when train set in a test-train
+    # split does not contain all classes
+    # Since this test uses LOO, at each iteration train set will not contain a
+    # class label
+    X = np.random.randn(10, 5)
+    y = np.arange(10)
+    clf = LinearSVC(C=1.0)
+    clf_prob = CalibratedClassifierCV(clf, method="sigmoid", cv=LeaveOneOut())
+    clf_prob.fit(X, y)
+    probs = clf_prob.predict_proba(X)
```

Comment:
@jnothman [This line](https://github.com/scikit-learn/scikit-learn/pull/7799/files/938ae64e74a145a6116149f408f46e9f4e47793a#diff-27280aea00f4713a0ee536b359e5a9deR295) should be the array of numbers corresponding to each class and not `np.arange(df.shape[1])`